Source/Core/DolphinQt2/GameList/GameList.cpp

```diff
@@ -34,9 +35,7 @@ static bool CompressCB(const std::string&, float, void*);
 GameList::GameList(QWidget* parent) : QStackedWidget(parent)
 {
   m_model = new GameListModel(this);
-  m_table_proxy = new QSortFilterProxyModel(this);
-  m_table_proxy->setSortCaseSensitivity(Qt::CaseInsensitive);
-  m_table_proxy->setSortRole(Qt::InitialSortOrderRole);
```

Comment:
This line needs to be kept in to allow sorting on the platform, region, file size, and state columns.